src/components/SelectMenu.tsx

```diff
@@ -1,10 +1,23 @@
+import { useModal } from '../hooks/modalHooks';
 import style from '../styles/SelectMenu.module.css';
+import AddFolderModal from './Modal/AddFolderModal';
+import DeleteLinkModal from './Modal/DeleteLinkModal';
 
 const SelectMenu = () => {
+  const {isOpen:deleteModalOpen, toggleModal:deleteToggleModal} = useModal();
+  const {isOpen:AddModalOpen, toggleModal:AddToggleModal} = useModal();
+
   return(
     <div className={`${style['select-menu--container']}`}>
-      <div className={`${style['menu']} ${style['delete-menu']}`}>삭제하기</div>
-      <div className={`${style['menu']} ${style['add-menu']}`}>폴더에 추가</div>
+      <button className={`${style['menu']} ${style['delete-menu']}`} 
+        onClick={(e) => {
+          e.preventDefault();
+          deleteToggleModal()}}>
```

Comment:
button요소의 type이 submit이 아니라서 preventDefault할만한 기본동작이 일어나지 않고, 아래처럼 사용해도 무방해보이네요
```suggestion
        onClick={deleteToggleModal}>
```

Reply:
아 음 링크 삭제 시, 해당 링크가 클릭되는 이슈가 있는 거 같은데, 이벤트 버블링이 의심되네요. propagation을 stop해줘야할거같읍니다

Reply:
제가 그걸 보고 버블링을 막으려고 stopPropagation을 했더니 적용이 안되더라고요. 그래서 preventDefault로 했더니 해결이 되서 그렇게 적어놨는데 무슨 차이가 있는지 잘 모르겠습니다. stopPropagation이 버블링을 막고 preventDefault가 기본 동작을 막는다는 걸로 알고 있는데 왜 이런 문제가 발생한 걸까요?

Reply:
아하
원인은 이 컴포넌트를 감싼 Card컴포넌트 구현 중, Link컴포넌트로 감싼 구현으로 인한 것으로 보입니다.
즉, a 태그안에 있는 모든 버튼들에서는 stopPropagation을 해도 이벤트전파가 막히지 않고 동작해서, a 태그가 동작하게 되는 것입니다.
이 설명은 [mdn문서](https://developer.mozilla.org/en-US/docs/Web/API/Event/stopPropagation) 에 있었습니다. 


> It does not, however, prevent any default behaviors from occurring; for instance, clicks on links are still processed. If you want to stop those behaviors, see the [preventDefault()](https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault) method.


지금처럼 `preventDefault`로 처리하신 게 지금 상황에서는 유일한 방벙이겠네요. 
이런 지식과 설명이 필요한 코드라면, 구현을 바꿔서 Link가 아닌 Button으로 구현을 바꾸는 것도 방법일 거 같네요